randovania/games/dread/json_data/header.txt

```diff
@@ -111,31 +111,222 @@ Templates
               Bomb and Water Bomb Jump (Beginner)
               Cross Bomb and Water Bomb Jump (Intermediate)
 
+* Open Charge Door:
+      Any of the following:
+          Shoot Charge Beam
+          Knowledge (Beginner) and Lay Power Bomb
+
+* Fight Silver Robot:
+      Any of the following:
+          Knowledge (Beginner) and Lay Power Bomb
+          All of the following:
+              Any of the following:
+                  # Methods of dodging
+                  Flash Shift or Combat (Beginner)
+              Any of the following:
+                  # Methods of fighting
+                  Shoot Plasma Beam
+                  Combat (Intermediate) and Shoot Charge Beam
+                  All of the following:
+                      # 47 Missiles are required, 1.5x leeway for trickless
+                      Shoot Missile
+                      Any of the following:
+                          Missiles ≥ 61
+                          Missiles ≥ 47 and Combat (Beginner)
```

Comment:
strict missile requirements should probably be higher than Beginner - maybe Intermediate? I'm also not seeing counter to refill missiles being accounted for here. in theory I'm pretty sure 1 missile is sufficient to beat a robot (and would obviously be hypermode) but something like 15ish + a combat level for the counter would be appropriate as well

Reply:
yeah i forgot to include the countering.